### ISSUE TITLE
Startup XmlConfiguration WARN on Arg threadpool

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#4631]
_Originally posted by @TezzerFX in https://github.com/eclipse/jetty.project/issues/4620#issuecomment-593572876_

There's a WARN message on startup about the threadpool ...

```
$ java -jar ../start.jar
2020-03-02 15:05:36.442:INFO::main: Logging initialized @381ms to org.eclipse.jetty.util.log.StdErrLog
2020-03-02 15:05:36.558:WARN:oejx.XmlConfiguration:main: Ignored arg: <Arg name="threadpool"><Ref refid="threadPool"/></Arg>
2020-03-02 15:05:36.696:WARN::main: demo test-realm is deployed. DO NOT USE IN PRODUCTION!
2020-03-02 15:05:36.698:INFO:oejs.Server:main: jetty-9.4.28-SNAPSHOT; built: 2020-03-02T18:07:43.942Z; git: f2a4c6b6150938c03022f29f3fa25bed5b7451f3; jvm 11.0.6+10
```

**Comment by joakime** [comment]
This is ultimately a bad warning, the argument isn't being ignored, and is being used properly.

It's just a later check for `<Arg>` is incorrectly generating a WARN level message when it shouldn't.

**Comment by sbordet** [comment]
Fixed by #4632.
